apps/prairielearn/src/lib/externalGrader.ts

```diff
@@ -1,22 +1,27 @@
-// @ts-check
 import * as async from 'async';
 import { z } from 'zod';
 import { logger } from '@prairielearn/logger';
 import * as sqldb from '@prairielearn/postgres';
 import * as Sentry from '@prairielearn/sentry';
-const assert = require('node:assert');
+import assert = require('node:assert');
+import type { EventEmitter } from 'node:events';
 
 import { config } from './config';
 import * as externalGradingSocket from './externalGradingSocket';
-const ExternalGraderSqs = require('./externalGraderSqs');
-const ExternalGraderLocal = require('./externalGraderLocal');
+import ExternalGraderSqs = require('./externalGraderSqs');
+import ExternalGraderLocal = require('./externalGraderLocal');
```

Comment:
Consider refactoring these to named exports so we can avoid `import ... = require(...)` syntax.

Reply:
Done in 28467d34d.